packages/nimble-components/src/anchor-menu-item/tests/anchor-menu-item.react.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { provideReactWrapper } from '@microsoft/fast-react-wrapper';
+import { AnchorMenuItem } from '..';
+
+const { wrap } = provideReactWrapper(React);
```

Comment:
I think we should avoid importing React and invoking provideReactWrapper in every file. Should just be done in one spot and exported file should just be
```ts
import { AnchorMenuItem } from '..';
import { wrap } from 'somewhere-probably-in-the-storybook-utils-folder';
export const NimbleAnchorMenuItem = wrap(AnchorMenuItem);
```